CONTRIBUTING.md

```diff
@@ -184,6 +185,16 @@ If you are adding a new feature, don't forget to edit the file
 `test/runnableTestSets.csv`. This file disables tests for features we
 have not yet implemented.
 
+If you want to check the performance of fontoxpath, you can pick some of
+qt3tests up randomly and run them via benchmark.js. As default, there is
+list. You can create the list which has 50 random qt3tests by running
+`npm run performance regenerate`. If you want to create a list which has
+less or more than 50 qt3tests, you can run `npm run performance regenerate 27`.
+Once you have a `test/runnablePerformanceTestNames.csv` file, you can run
```

Comment:
is this checked in? when should this be checked in?

Reply:
I added it to gitignore